reflex/components/radix/primitives/drawer.py

```diff
@@ -230,11 +231,17 @@ def _get_style(self) -> dict:
         return self.style
 
 
-drawer_root = DrawerRoot.create
-drawer_trigger = DrawerTrigger.create
-drawer_portal = DrawerPortal.create
-drawer_content = DrawerContent.create
-drawer_overlay = DrawerOverlay.create
-drawer_close = DrawerClose.create
-drawer_title = DrawerTitle.create
-drawer_description = DrawerDescription.create
+class Drawer(SimpleNamespace):
```

Comment:
I think we can make a subclass of SimpleNamespace such that we can define these like:

```python
class Drawer(NamespaceSubclass):
    root = DrawerRoot
    trigger = DrawerTrigger
    ...
```

Reply:
that would break static analysis and autocomplete i think. i tried making a fancy class first

Reply:
to be clear, i'm not opposed to the idea per-se, but it might be trickier to get the typing right and would definitely need some help from `pyi_generator`.

i'd prefer to go with the simpler, expliciter route for now, but after we get this all out the door, i'm not opposed to a refactor or cleanup of the namespace classes